__tests__/app.test.js

```diff
@@ -99,3 +99,43 @@ describe('GET /api/articles/:article_id', () => {
 			})
 	})
 })
+
+describe('/api/articles/:article_id', () => {
+	test('PATCH:200 responds with updated article, with incremented votes property', () => {
+		const articleUpdate = {
+			inc_votes: 2,
+		}
+		return request(app)
+			.patch('/api/articles/1')
+			.send(articleUpdate)
+			.expect(200)
+			.then((response) => {
+				const article = response.body.article
+
+				expect(article).toEqual({
+					article_id: 1,
+					title: 'Living in the shadow of a great man',
+					topic: 'mitch',
+					author: 'butter_bridge',
+					body: 'I find this existence challenging',
+					created_at: '2020-07-09T20:11:00.000Z',
+					votes: 102,
+					article_img_url:
+						'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+				})
+			})
+	})
+	test('PATCH:400 responds with an error message when provided with a malformed request body', () => {
+		const articleUpdate = {}
+
+		return request(app)
+			.patch('/api/articles/1')
+			.send(articleUpdate)
+			.expect(400)
+			.then((response) => {
+				const message = response.body.message
+
+				expect(message).toBe('Bad request')
+			})
+	})
+})
```

Comment:
Nice work so far :) 

We ill need the following tests before we can merge:
- 404 article not found (i.e. article_id requested = 9999)
- 400 invalid id (i.e. article_id requested = "banana")
- 400 inc_vote not an integer (i.e. const articleUpdate = {inc_votes: "banana"}